src/utils/tree.js

```diff
@@ -13,29 +13,33 @@ var tf =
     },
 
     showTree: function(newickStr) {
-      var newick = this.require("biojs-io-newick");
+      var newick = require("biojs-io-newick");
```

Comment:
Not to forget the async logic here that allows development and production bundles and does most of the logic:

https://github.com/wilzbach/msa/blob/master/src/g/package.js